pkg/apis/componentconfig/types.go

```diff
@@ -111,6 +111,9 @@ type KubeSchedulerConfiguration struct {
 	// Indicate the "all topologies" set for empty topologyKey when it's used for PreferredDuringScheduling pod anti-affinity.
```

Comment:
This is just a general comment, not necessary for this PR, but you should move the scheduler config to its own API group. See https://docs.google.com/document/d/1FdaEJUEh091qf5B98HM6_8MS764iXrxxigNIdwHYW9c.